src/sass/style.scss

```diff
@@ -35,88 +35,95 @@ html {
     box-sizing: inherit;
 }
 
-.info {
+.about {
+    margin-left: $spacing-m;
+    margin-right: $spacing-m;
+}
+
+.about_title {
     border-bottom: solid $accent-border-colour;
     color: $primary-text-colour;
     font-size: 1.1em;
     margin-bottom: $spacing-l;
     width: 3.5rem;
 }
 
-.about {
-    margin-left: $spacing-m;
-    margin-right: $spacing-m;
-}
-
-.about--info {
+.about_header {
     color: $primary-text-colour;
     font-size: 3.2em;
     font-weight: bold;
     line-height: 1.11em;
     margin-bottom: $spacing-m;
 }
 
-.about--links {
+.about_info {
     color: $secondary-text-colour;
     font-size: 1.6em;
     line-height: 1.5em;
     margin-bottom: $spacing-l;
 }
 
-.links {
-    color: $info-links-colour;
+.about_links {
+    color: $about_links-colour;
     transition: background-color 0.2s;
 }
 
-.links:hover {
+.about_links:hover {
     background-color: $hover-highlight-colour;
 }
 
-.aside-img {
+.aside_img {
     display: block;
     height: 25vh;
     margin-bottom: $spacing-s;
     object-fit: cover;
     width: 100vw;
 }
+
 .icons {
     visibility: hidden;
 }
-.github,
-.twitter {
+
+.icons_github,
+.icons_twitter {
     height: 2.25em;
     visibility: hidden;
     width: 2.25em;
 }
-.twitter {
+
+.icons_twitter {
     fill: #1da1f2;
 }
-.twitter:hover {
+
+.icons_twitter:hover {
     fill: lightblue;
 }
-.github:hover {
+
+.icons_github:hover {
     fill: darkgrey;
 }
+
 // Adjusts layout between mobile & tablet screen sizes
 @media screen and (min-width: 26.1rem) and (max-width: 46rem) {
```

Comment:
- Use two unserscores becacause is an Element (in BEM).
- Wired media query, right? 26.1rem? Thing about what I told you :)

Reply:
Re: Media query, should I be thinking more about device orientation rather than width here?

Reply:
The problem is that you should not use REMS in MQ and the decimal value (26.1 is kind of wired for a mq)

Reply:
Mq updated